src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -389,6 +409,10 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
     )
   }
 
+  private selectedBid(): Bid {
+    return this.props.increments[this.state.selectedBidIndex]
```

Comment:
We never refresh the increments on these "internal" views, so this shouldn't be a problem. However, we could add logic to avoid accessing arrays with indices that are out of bounds.